Makefile

```diff
@@ -9,14 +9,18 @@ restylers.yaml: */info.yaml
 	./build/restyler-meta get "$*" image | ./build/push-image
 	echo > $@
 
-%/Dockerfile.tested: %/Dockerfile.built
-	cram "test/$*.t"
+%/Dockerfile.tested: %/Dockerfile.built test/%.t
```

Comment:
:man_facepalming:

Reply:
the `.built` has a similar bug but it's harder to quantify -- since technically any file change in the `./${thing}/` subdir could change the build -- I didn't attempt to fix that one though -- make isn't exactly a good fit for that

Reply:
Yeah, I think because in the most common case we should strive for just a `Dockerfile` required to produce a Restyler, it's acceptable to have that wart.